core/src/main/java/bio/terra/pearl/core/service/survey/SurveyResponseService.java

```diff
@@ -131,6 +135,16 @@ private List<Answer> getReferencedAnswers(Enrollee enrollee, Survey survey) {
         return answers;
     }
 
+    //if the cutoff time has passed, create a new task and response
+    private boolean shouldCreateNewLongitudinalTaskAndResponse(Integer createNewResponseAfterDays, Instant surveyResponseLastUpdatedAt) {
+        if(createNewResponseAfterDays == null) {
+            return false;
+        }
+        Instant cutoffTime = ZonedDateTime.now(ZoneOffset.UTC)
+                .minusDays(createNewResponseAfterDays).toInstant();
+        return surveyResponseLastUpdatedAt.isBefore(cutoffTime);
+    }
+
```

Comment:
I was thinking about this ticket and another point of trickiness that I think we need to account for is - what happens if you want to edit a previous response?

Like, let's say you have 5 responses, and you realize the second one is incorrect - if you edit that second one, I'm assuming it should edit in place. But if you attempt to edit the _latest_ one, it should make a new one.

I dunno if that's the right approach, but it feels right to me.